Dockerfile

```
@@ -42,7 +42,7 @@ ADD decidim-bulletin_board-app/Gemfile.lock /code/tmp/Gemfile.lock
 RUN cd /code/tmp && bundle install
 
 # Add local python dependencies
-ADD voting_schemes/election_guard/python-wrapper /code/voting_schemes/election_guard/python-wrapper
+ADD voting_schemes//python-wrapper /code/voting_schemes/electionguard/python-wrapper
```

I'm getting this error when deploying, looks like it is related to this
```
Step 23/33 : ADD voting_schemes//python-wrapper /code/voting_schemes/electionguard/python-wrapper
ADD failed: stat /var/lib/docker/tmp/docker-builder003347322/voting_schemes/python-wrapper: no such file or directory

```
```suggestion
ADD voting_schemes/electionguard/python-wrapper /code/voting_schemes/electionguard/python-wrapper
```